src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+html,
+body {
+  font-family: "Roboto", sans-serif;
+  /* font-weight: 500; */
+  font-size: 12px;
+  margin: 0;
+}
```

Comment:
margin for the body
and font for the HTML
No need to join

Reply:
perfect already done.